scripts/k8s/deployer.sh

```diff
@@ -31,9 +32,10 @@ Options:
   --disk          <size>            Add disk of this size to each worker node.
   --dry-run                         Don't do anything, just output steps.
   --hugepages     <num>             Add <num> 2MiB hugepages (Default: $HUGE_PAGES).
-  --label                           Label worker nodes with the io-engine selector.
-  --install-zfs                     Install ZFS utilities.
-  --install-lvm                     Install LVM utilities.
+  --label-io-engine                 Label worker nodes with the io-engine selector.
+  --setup-zfs                       Install ZFS utilities.
+  --setup-lvm                       Install LVM utilities and load required modules.
+  --setup-mayastor                  Setup hugepages 
```

Comment:
Just a suggestion, maybe drop the prefix
Then on the help message, we also install and load kernel modules.

```suggestion
  --zfs                       Install ZFS utilities.
  --lvm                      Install LVM utilities and load required modules.
  --mayastor            Setup pre-requisites, install and load required modules.
```